clustered/client/src/test/java/org/ehcache/clustered/client/internal/service/ConnectionClosedTest.java

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
This is not testing the lease right? Just that the connection still works. Can you test lease expiration and renewal?

Reply:
Lease expiry and renewal are already tested the lease code.

Reply:
And you are super sure we don't need integration tests at the ehcache level?

Reply:
In integration test, as of now I don't see any way to control lease expiry/ renewal stuff. Since lease mechanism is as good as a normal connection timeout at the entity layer, since lease is hidden right?